Apps/Admin/Server/Services/CsvExportService.cs

```diff
@@ -116,19 +116,19 @@ public Stream GetUserFeedback()
         }
 
         /// <inheritdoc/>
-        public Stream GetYearOfBirthCounts(DateTime startDate, DateTime endDate, int timeOffset)
+        public Stream GetYearOfBirthCounts(string startPeriod, string endPeriod, int timeOffset)
         {
-            int offset = timeOffset * -1;
-            TimeSpan timeSpan = new(0, offset, 0);
-            DateTime startDateUtc = startDate.Date.Add(timeSpan);
-            startDateUtc = DateTime.SpecifyKind(startDateUtc, DateTimeKind.Utc);
-            DateTime endDateUtc = endDate.Date.Add(timeSpan).AddDays(1).AddMilliseconds(-1);
-            endDateUtc = DateTime.SpecifyKind(endDateUtc, DateTimeKind.Utc);
-            IDictionary<string, int> yobCounts = this.userProfileDelegate.GetLoggedInUserYearOfBirthCounts(startDateUtc, endDateUtc);
+            TimeSpan ts = new(0, timeOffset, 0);
```

Comment:
I thought the offset would need to be multiplied by -1 (to match the behaviour of the DashboardService), but it looks like the offset calculated on AnalyticsPage.razor.cs is multiplied by -1 before it is sent to the back-end, while the offset calculated on DashboardPage.razor.cs is not.

I agree with @BrianMaki that it would be better if we could do away with the offset entirely when we are passing dates.

Reply:
Okay; I'm fine to merge it as-is now and if we can figure out how to replace the offset, we can do that later.